### ISSUE TITLE
No additional loading triggered when window can accommodate initial files without scrolling.

### ISSUE THREAD

**Issue by ianwarnold** [filebrowser/filebrowser#1238]
**Description**
On a large display (1440p) in a browser with default font and zoom settings, in a folder with a lot of files, a large browser window can accommodate all of the 50 files that are loaded initially without any need for scrolling. Because loading of additional files seems to be triggered by the scroll bar hitting the bottom, File Browser will not load additional files unless the window is made smaller so that scrolling occurs.

**Expected behaviour**
I expected File Browser to go beyond the initial 50 files and continue to load files until the screen is filled (that is, until scrolling is necessary to access the overflow).

**What is happening instead?**
File Browser only loads 50 files and won't load more unless the window is reduced in size. No log messages are emitted.

**Additional context**
See video of behavior here: https://www.youtube.com/watch?v=-FmH4acRquo

**How to reproduce?**

1. Create a folder with a bunch of files; e.g. `for LETTER1 in {A..Z} ; do for LETTER2 in {A..Z} ; do touch ${LETTER1}${LETTER2} ; done ; done`
2. Launch File Browser with the root dir set to the folder containing the many files.
3. In a large browser window (e.g. maximized at 1440p), access the File Browser web GUI and note that files beyond the initial 50 are not loaded.

**Files**
N/A.

**Comment by ianwarnold** [comment]
Seems like something might need to be added [in here](https://github.com/filebrowser/filebrowser/blob/26d62e411716a5eb9a5a703e47484cfb3fbf3bd0/frontend/src/components/Search.vue#L138-L144)? I'd offer to do it myself but I really know very little about web frontends and literally nothing at all about Vue.

**Comment by WeidiDeng** [comment]
It's [this line](https://github.com/filebrowser/filebrowser/blob/master/frontend/src/components/files/Listing.vue#L326). You can try do this comparison in [mounted](https://github.com/filebrowser/filebrowser/blob/master/frontend/src/components/files/Listing.vue#L164).

**Comment by ianwarnold** [comment]
Did some more experimentation, this time on my phone. I did 3 experiments:

1. A folder whose content is other folders only
2. A folder whose content is files only
3. A folder whose content is a mixture of files and folders.

Both case 1 and case 2 demonstrated nondeterministic loading failures--that is, sometimes loading would work, sometimes loading would fail. In both cases, it was not a serious problem, because scrolling up a little bit and then back down would always trigger a load, even if it had failed to trigger previously.

Case 3 is a more severe failure--50 folders load initially and 0 files, and no more loading can be triggered.

I got video of all 3 cases:

1. https://www.youtube.com/watch?v=wjOuefb2XFw
2. https://www.youtube.com/watch?v=_J9IhQ3lF8s
3. https://www.youtube.com/watch?v=rujnEm15B4w
